public/app/features/alerting/unified/components/contact-points/utils.ts

```diff
@@ -30,6 +31,9 @@ export function isProvisioned(contactPoint: GrafanaManagedContactPoint) {
 
 // TODO we should really add some type information to these receiver settings...
 export function getReceiverDescription(receiver: ReceiverConfigWithMetadata): ReactNode | undefined {
+  if (!receiver.settings) {
```

Comment:
needed as the new endpoint for receivers list does not include this field in the response

Reply:
and it's not complaining 🤷‍♀️. I think RTKQ it's not able to check if this response is correct regarding the type:

```
    getContactPointsList: build.query<GrafanaManagedContactPoint[], void>({
      query: () => ({ url: '/api/v1/notifications/receivers' }),
    }),
```